Plugins/IngameDebugConsole/Scripts/DebugLogManager.cs

```diff
@@ -601,7 +601,7 @@ private void OnEnable()
 
 			if( receiveLogcatLogsInAndroid )
 			{
-#if !UNITY_EDITOR && UNITY_ANDROID
+#if !UNITY_EDITOR && UNITY_ANDROID && UNITY_ANDROID_JNI
```

Comment:
Thank you for the PR! Can you log a warning here if `UNITY_ANDROID_JNI` is not defined:

> Android JNI module must be enabled in Package Manager for "Receive Logcat Logs In Android" to work.